hw/ip/otbn/README.md

```diff
@@ -27,17 +47,24 @@ fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ip:otbn_top_sim
 
 It includes functionality to set the initial DMem and IMem contents. The start
 address is hard coded to 0. Modify the `ImemStartAddr` parameter in
-`./dv/verilator/otbn_top_sim.sv` to change this. The simulation is run as
+`./dv/verilator/otbn_top_sim.sv` to change this. Combined with the build script
+described above a single assembly file can be built and run on the simulation as
 follows:
 
 ```sh
+# Build ./example_dir/prog.S
+./hw/ip/otbn/util/build.sh ./example_dir/prog.S ./example_dir/prog_bin
+
+# Run the resulting binary on the OTBN standalone simulation
 ./build/lowrisc_ip_otbn_top_sim_0.1/sim-verilator/Votbn_top_sim -t \
-  --meminit=imem,imem_contents.vmem --meminit=dmem,dmem_contents.vmem
+  --meminit=imem,./example_dir/prog_bin_imem.elf \
+  --meminit=dmem,./example_dir/prog_bin_dmem.elf
```

Comment:
Could we change this example to be one of the code snippets? It's nice to be able to copy-paste the example directly into the terminal and run it. (Something that I've got annoyed with when running Ibex simple_system in the past)

Reply:
This was left as is in the commit that adds the build script then switched to the smoke test .S in the commit that adds the smoke test. I used the smoke test as modexp.S will need some dmem contents to process plus uses unimplemented instructions and the psuedo-ops.S seemed a little too uninteresting.